api/src/main/java/org/openmrs/module/fhir2/providers/r3/MedicationRequestFhirResourceProvider.java

```diff
@@ -11,17 +11,28 @@
 
 import javax.validation.constraints.NotNull;
 
-import ca.uhn.fhir.rest.annotation.IdParam;
-import ca.uhn.fhir.rest.annotation.Read;
+import ca.uhn.fhir.rest.annotation.*;
```

Comment:
Don't use `*` imports

Reply:
Oooh yes...though I have noticed a couple of other resources when its done.